### PR TITLE
Donut controller

### PR DESCRIPTION
**Pull request by nikstojkov**
Run migration files, DB created,

### CODE REVIEW

File: app/controllers/donut_controller.rb [hunk 1]
```diff
@@ -2,4 +2,8 @@ class DonutController < ApplicationController
   def index
     @donuts = Donut.all
   end
+
+  def show
+    @donut = Donut.find
```
Comment (sulphite) [inline app/controllers/donut_controller.rb:7]: need to add params[:id] in the find method

File: app/models/donut.rb [hunk 2]
```diff
@@ -1,2 +1,3 @@
 class Donut < ApplicationRecord
+  belongs_to :user
```
Comment (sulphite) [inline app/models/donut.rb:2]: can we also add that it has many bookings

**Review by sulphite — CHANGES_REQUESTED** [review]
some small changes

**Comment by JDS478** [comment]
Schema is looking good, just bookings controller and references in schema needed

**Pull request merged**